### PR TITLE
simplify installation procedure

### LINKED ISSUE #50: Simplify the installation procedure
**Describe the feature**

**Motivation**
mmocr installation needs to compile c++ and cuda code.
We'd better moving the c++/cuda functions or ops to mmcv. 

1.   assign_pixels  in panet
2.   estimate_text_confidence in panet
3.  get_pixel_num in panet
4.  pse in psenet
5.  rroi align

**Related resources**
Nil

**Additional context**
Nil

If you would like to implement the feature and create a PR, please leave a comment here and that would be much appreciated.

### PR DESCRIPTION
**Pull request by jeffreykuang**
Fix #50 

- move panet, and psenet postprocessing to mmcv 1.3.4
- speed up psenet postprocessing so that it can detect dense texts
- mv rroialign to mmcv
- modify dependence and setup.py accordingly.

### CODE REVIEW

File: configs/textdet/panet/panet_r18_fpem_ffm_600e_icdar2015.py [hunk 1]
```diff
@@ -79,7 +79,7 @@
 ]
 data = dict(
     samples_per_gpu=8,
-    workers_per_gpu=2,
+    workers_per_gpu=0,
```
Comment (jeffreykuang) [inline configs/textdet/panet/panet_r18_fpem_ffm_600e_icdar2015.py:82]:
```suggestion
    workers_per_gpu=2,
```

File: docs/changelog.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
 # Changelog
+## v0.2.0 (16/5/2021)
+
+**Highlights**
+
+- MMOCR is compiling-free via moving textdet postprocessing and ops to mmcv 1.3.4 or later.
+- Add a new OCR downstream task-NER.
+- Add two new text detection methods: DRRG and FCENet.
+- Add end-to-end demo.
```
Comment (jeffreykuang) [inline docs/changelog.md:9]:
```suggestion
- Add an end-to-end demo.
```

File: mmocr/models/textdet/postprocess/__init__.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
-from .wrapper import decode
+from .wrapper import decode  # NOQA
```
Comment (jeffreykuang) [inline mmocr/models/textdet/postprocess/__init__.py:1]:
```suggestion
from .wrapper import decode
```

File: mmocr/version.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
 # Copyright (c) Open-MMLab. All rights reserved.
 
-__version__ = '0.1.0'
+__version__ = '0.2.0'
```
Comment (innerlee) [inline mmocr/version.py:3]: We can leave version bump to a separate PR.

Comment (jeffreykuang) [inline mmocr/version.py:3]:
```suggestion
__version__ = '0.1.0'
```

**Comment by innerlee** [comment]
This file is too large. Can it be optimized to, say <600K?

image

**Review by innerlee — APPROVED** [review]

**Review by cuhk-hbsun — APPROVED** [review]

**Comment by seekingdeep** [comment]
@innerlee @jeffreykuang in-order to optimize an image without resizing or loosing much quality, you can use mozjpeg:
```
sudo apt install libjpeg-turbo-progs
cjpeg -quality 80 input.jpg > output.jpg
```
